test/pytest/test_distributed_inference_handler.py

```diff
@@ -26,7 +26,7 @@
 
 
 @pytest.mark.skipif(
```

Comment:
You can break apart these lines to make it easier to read

```
pytest.mark.skipIf(condition_a)
pytest.mark.skipIf(condition_b)
test_function()
```

I had to triple read the not condition and where the parenthesis was to make sure there wasn't some bug that would run this on CPU

Would also be great to make sure this test gets skipped in the CPU regression tests

Reply:
Thanks Mark, sure, just testing now if solve the issue the would fix regression_gpu.yaml back to run nightlies.